Adds Superhuman Physiology, a martial art used by clings.

hal9000PR opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do
Adds a new martial art, used by clings it can be purchased for 3 DNA points
this martial art has a couple properties,

If you are examined by someone within one tile of you (similarly to shadowling thralls) they will get an examine message denoting that you have this martial art. `Their muscles seem to buldge and shift in an inhuman manner.`

**basic attacks**

harm intent: the cling uses claws to lash at their target dealing 12 damage per swing

disarm:  works slightly more often, and knocks the target back one tile when successful doing a little bit of damage

grab: instantly aggressive grab

**combos** 

Hammer fist  
Steps: Disarm, Harm, Harm

Deals 20 damage, stuns the target for 2-4 seconds, and deals 30 stamina damage deals more damage if the target is already stunned. Doesn't stun targets who are already stunned.

*We morph our fist into a bone hammer and slam our target to the ground, deals double damage to already stunned targets.*

Piston Kick
Steps: Disarm, Grab

deals little damage but launches the target 10 tiles away and confuses them, good for disengaging. also damages one of your legs a small amount

*We use explosive chemical reactions to kick someone with such a force that it sends them flying*

Foot Skewer
Steps: Harm, Grab

deals damage to your targets foot, and slows them for 10 seconds, good opening move, makes clicking on people easier.

*We reform our foot into a knife and stab it through our opponent's foot, crippling their mobility.*

Brutal Barrage
Steps: Harm, Disarm, Grab, Harm

Punches your opponent in the face really hard, deals more damage the more damage the target has already taken. good finishing move.

*We unleash a flurry of swift blows that completely decimate a weakened opponent.*
## Why It's Good For The Game
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

Currently, clings are a little lacklustre. you are better off not using changeling abilities because of their KOS status. This adds a little more variety to clings, instead of it just being "hurr durr armblade" there is more depth to it than just clicking until horizontal.

Personally I think this suits the cling aesthetic as well. Using their advanced biology and memories/skills of those who they have once absorbed to form a complex system of attacks, growing claws and knives, using their heightened strength to knock people down.

The damage and stun numbers are entirely changeable if you guys think they are too high or too low.

## Changelog
:cl:
add: adds a new martial art purchasable by clings for 3 DNA points, Superhuman physiology. Basic punches are claws, dealing high damage. Disarms are more likely to work and knockback and damage the target. Being examined within one tile gives the examiner a message denoting that you have the ability, similarly to shadowling thralls. you get 4 combos aswell, hammer fist - a high damage stunning combo. Piston kick - a knockback combo. Foot skewer - a light damage combo which slows the target. Brutal barrage - a combo that does more damage the more damaged the target is.

/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

S34NW commented on code/game/gamemodes/changeling/powers/superhuman_physiology.dm:
```diff
@@ -0,0 +1,16 @@
+/datum/action/changeling/superhuman
+	name = "Superhuman Physiology"
+	helptext = "We may toggle our superhuman biology on and off to avoid detection."
```
```suggestion
	helptext = "We can toggle our superhuman biology on and off to avoid detection."
```

hal9000PR replied:
wait.... thats not even something you can do. description is just wrong here

hal9000PR replied:
Fixed

S34NW commented on code/game/gamemodes/changeling/powers/superhuman_physiology.dm:
```diff
@@ -0,0 +1,16 @@
+/datum/action/changeling/superhuman
+	name = "Superhuman Physiology"
+	helptext = "We may toggle our superhuman biology on and off to avoid detection."
+	desc = "We combine the fighting knowledge of the hivemind and our enhanced biological strength to form a potent martial art that makes use of our unique abilty to create weapons. Costs 30 chemicals to toggle"
```
```suggestion
	desc = "We combine the fighting knowledge of the hivemind and our enhanced biological strength to form a potent martial art that makes use of our unique abilty to create weapons. Costs 30 chemicals to toggle."
```

S34NW commented on code/modules/martial_arts/combos/superhuman/piston_kick.dm:
```diff
@@ -0,0 +1,17 @@
+/datum/martial_combo/superhuman/piston_kick
+	name = "Piston Kick"
+	explaination_text = "We use explosive chemical reactions to kick someone with such a force that it sends them flying"
```
```suggestion
	explaination_text = "We use explosive chemical reactions to kick someone with such a force that it sends them flying."
```

S34NW commented on code/modules/martial_arts/combos/superhuman/piston_kick.dm:
```diff
@@ -0,0 +1,17 @@
+	user.apply_damage(5, BURN, pick(BODY_ZONE_R_LEG, BODY_ZONE_L_LEG)) //your leg is literally exploding like a piston
+	var/atom/throw_target = get_edge_target_turf(target, user.dir)
+	target.throw_at(throw_target, 10, 14, user)
+	target.visible_message("<span class='warning'>[user]'s lower leg explosively extends into [target] sending them flying!</span>", \
```
```suggestion
	target.visible_message("<span class='warning'>[user]'s lower leg is explosively propelled into [target], sending them flying!</span>", \
```

Samman166 commented:
This sounds cool as fuck and I want it. On one hand I think it might make armblade a bit obsolete but it still does double the damage as this and is useful just to pry doors/remove heads

S34NW commented on code/modules/martial_arts/combos/superhuman/piston_kick.dm:
```diff
@@ -0,0 +1,17 @@
+	var/atom/throw_target = get_edge_target_turf(target, user.dir)
+	target.throw_at(throw_target, 10, 14, user)
+	target.visible_message("<span class='warning'>[user]'s lower leg explosively extends into [target] sending them flying!</span>", \
+					"<span class='userdanger'>[user] lower leg explosively extends into you! sending you flying and disorintating you!</span>")
```
```suggestion
					"<span class='userdanger'>[user]'s lower leg is explosively propelled into you! Sending you flying and disorienting you!</span>")
```

S34NW commented on code/modules/martial_arts/superhuman.dm:
```diff
@@ -0,0 +1,51 @@
+	to_chat(usr, "<b><i>We scour the memories of all those we have absorbed, remembering their fighting styles... </i></b>")
+
+/datum/martial_art/superhuman/explaination_footer(user)
+	to_chat(user, "<b><i>In addition, by having your throw mode on when being attacked, we enter an active defense mode where we block and sometimes even counter attacks done to you.</i></b>")
```
```suggestion
	to_chat(user, "<b><i>In addition, by having our throw mode on when being attacked, we enter an active defense mode where we block and sometimes even counter melee attacks done to us.</i></b>")
```

S34NW reviewed (commented):
Really good work, just some small grammar things.

LordShekelstein commented on code/modules/martial_arts/combos/superhuman/brutal_barrage.dm:
```diff
@@ -0,0 +1,17 @@
+/datum/martial_combo/superhuman/brutal_barrage
+	name = "Brutal Barrage"
+	explaination_text = "We unleash a flurry of swift blows that completely decimate a weakened oppenent."
```
```suggestion
	explaination_text = "We unleash a flurry of swift blows that completely decimates a weakened oppenent."
```
haha u luv martial orts lmaoooo

LordShekelstein commented on code/modules/martial_arts/combos/superhuman/foot_skewer.dm:
```diff
@@ -0,0 +1,16 @@
+/datum/martial_combo/superhuman/foot_skewer
+	name = "Foot Skewer"
+	explaination_text = "We reform our foot into a knife and stab it through our opponent's foot, crippling their mobility."
```
```suggestion
	explaination_text = "We reform our foot into a serrated blade and stab it deep into our opponent's foot eviscerating tendons and ligaments, crippling their mobility.
."
```
This is mostly fluff text to make it more epic. Your grammar and spelling was correct.

LordShekelstein replied:
Also my quote is in the wrong spot, heck.

LordShekelstein replied:
Mhm, apologies, I had a major migraine whilst writing my changes.

SabreML commented on code/modules/martial_arts/superhuman.dm:
```diff
@@ -0,0 +1,51 @@
+
+/datum/martial_art/superhuman/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	MARTIAL_ARTS_ACT_CHECK
+	var/obj/item/grab/G = D.grabbedby(A, 1)
```
```suggestion
	var/obj/item/grab/G = D.grabbedby(A, TRUE)
```

SabreML commented on code/modules/martial_arts/superhuman.dm:
```diff
@@ -0,0 +1,51 @@
+/datum/martial_art/superhuman/disarm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	MARTIAL_ARTS_ACT_CHECK
+	A.do_attack_animation(D, ATTACK_EFFECT_DISARM)
+	if(prob(60))
+		if(!D.stat && !D.IsWeakened())
+			D.visible_message("<span class='warning'>[A] slams into [D], knocking them to the ground!</span>", \
+								"<span class='userdanger'>[A] slams into you, knocking you to the ground!</span>")
+			playsound(get_turf(D), 'sound/weapons/punchmiss.ogg', 50, 1, -1)
+			D.apply_damage(5, BRUTE)
+			D.Weaken(2)
```
60% chance to stun for 4 seconds every time you disarm seems pretty high to me.

hal9000PR replied:
This is fairly similar to standard disarms. Also it is a 2-4 second stun rather than 4 seconds. I'm unsure on how to make the disarms better without completely reworking disarms in general.

SabreML commented on code/modules/martial_arts/superhuman.dm:
```diff
@@ -0,0 +1,51 @@
+				var/atom/throw_target = get_edge_target_turf(D, A.dir)
+				D.throw_at(throw_target, 1, 14, A)
+	else
+		D.visible_message("<span class='danger'>[A] attempted to disarm [D]!</span>", "<span class='userdanger'>[A] attempted to disarm [D]!</span>")
```
```suggestion
		D.visible_message("<span class='danger'>[A] attempted to disarm [D]!</span>", "<span class='userdanger'>[A] attempted to disarm you!</span>")
```
?

SabreML commented on code/modules/martial_arts/combos/superhuman/brutal_barrage.dm:
```diff
@@ -0,0 +1,17 @@
+	user.do_attack_animation(target, ATTACK_EFFECT_SMASH)
+	playsound(get_turf(target), 'sound/weapons/punch1.ogg', 75, 1, -1)
+	if(target.health <= 20)
+		target.visible_message("<span class='warning'>[user]'s arm swings, enlarging before impact, striking [target]'s face with great force, knocking them out!</span>", \
+						  "<span class='userdanger'>[user] slams you in the face, knocking you out!</span>")
+		target.SetSleeping(10) //KO
+	else
+		target.visible_message("<span class='warning'>[user]'s arm swings, enlarging before impact, striking [target]'s face with great force!</span>", \
+						  "<span class='userdanger'>[user] slams you in the face!</span>")
```
```suggestion
		target.visible_message("<span class='warning'>[user]'s arm swings, enlarging before impact, striking [target]'s face with great force, knocking them out!</span>",
						  "<span class='userdanger'>[user] slams you in the face, knocking you out!</span>")
		target.SetSleeping(10) //KO
	else
		target.visible_message("<span class='warning'>[user]'s arm swings, enlarging before impact, striking [target]'s face with great force!</span>",
						  "<span class='userdanger'>[user] slams you in the face!</span>")
```
You don't actually need the backslashes here after the `visible_message()`. (Even though they're _everywhere_ in the codebase.)
Those would only apply if you wanted to continue the string onto the next line without cutting it. In this case you've finished with the first argument so the comma makes it perfectly fine for the second argument to be on a new line.
Same thing in the other files too.

SabreML reviewed (commented):
👀

Wojz commented on code/modules/martial_arts/combos/superhuman/hammer_fist.dm:
```diff
@@ -0,0 +1,17 @@
+/datum/martial_combo/superhuman/hammer_fist
+	name = "Hammer Fist"
+	explaination_text = "We morph our fist into a bone hammer and slam our target to the ground, deals double damage to already stunned targets."
```
I think it might be even better as "The damage is doubled against already stunned targets."

Wojz commented:
100% block on an antag with virtually unlimited EMP might be a tad strong, the baton is frequently the way to deal with the buggers

Rurik123 commented:
> grab: instantly aggressive grab

This'll be fun.

Though it should be noted, this will make mindswaps even easier. A literal naked guy can instant grab HoS walking down the hallway, swapping bodies before help can arrive. I don't mind too much since usually victim gets a call of help and bystanders are almost always around, but figured it should be said.

procdrone commented:
im little concerned for ability to reliably stun & deal massive damage for free but looks cool otherwise

Wojz commented:
> > grab: instantly aggressive grab
> A literal naked guy can instant grab HoS walking down the hallway, swapping bodies before help can arrive. 

Did you mean: _**Tentacle**_?

Dumbdumn5 commented:
I love this, but what're the stun times in plain english because I am n o t a code wizard and don't quite understand setting somebody's 'sleep' to ten. Is it ten ingame seconds of unconsciousness? Because that's asinine in terms of power and combat changes are trying to r e d u c e stuns. 

When looking at numbers in combat, one second is a minor hindrance, three seconds is a serious issue, and five seconds is an eternity. Considering this martial art seems to be about zoning and combo-mealing people into oblivion, keeping the times quick rather than very long should help. That ten tile fling and confuse should do enough to stun people on its own, just sorta curious on the hammer stun time in seconds is all.

Kyep reviewed (commented):
Frankly, this seems a bit OP. Rather like arm blade, except mostly better, and people can't tell you have it (and thus, are a cling) from looking at your sprite.

hal9000PR commented:
Right, imma reduce some of the stun times but maybe I'll substitute some of them with stamina damage? So initially the stuns are short and you can escape/fight back. But after a while of using combos on someone they get stunned for much longer? 

 @Dumbdumn5 `weaken(3)` means it will stun someone for 3 cycles, meaning 4-6 seconds, so basically a telebaton. The sleep means 18-20 seconds which admittedly could be a little long, however it is a lengthy combo which involves a grab, making it harder to pull off because you cannot do it on the move. Once you grab them you are slowed, so having a long difficult combo to pull off which does big damage and if you are hurt already knocks you out could be okay. However if you feel any of this is too much i can: decrease the stun times, decrease the damage values, increase the DNA cost.

farie82 commented on code/game/gamemodes/changeling/powers/superhuman_physiology.dm:
```diff
@@ -0,0 +1,15 @@
+	name = "Superhuman Physiology"
+	desc = "We combine the fighting knowledge of the hivemind and our enhanced biological strength to form a potent martial art that makes use of our unique abilty to create weapons."
+	dna_cost = 3
+	req_human = 0
```
```suggestion
	req_human = FALSE
```

farie82 commented on code/modules/martial_arts/combos/superhuman/brutal_barrage.dm:
```diff
@@ -0,0 +1,17 @@
+	else
+		target.visible_message("<span class='warning'>[user]'s arm swings, enlarging before impact, striking [target]'s face with great force!</span>",
+						  "<span class='userdanger'>[user] slams you in the face!</span>")
+	target.apply_damage(target.getBruteLoss() * 0.75, BRUTE, BODY_ZONE_HEAD)
```
This can do an absurd amount of damage if left uncapped. Just a note

farie82 commented on code/modules/martial_arts/combos/superhuman/brutal_barrage.dm:
```diff
@@ -0,0 +1,17 @@
+	explaination_text = "We unleash a flurry of swift blows that completely decimates an oppenent in a critical state."
+	steps = list(MARTIAL_COMBO_STEP_HARM, MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_HARM)
+
+/datum/martial_combo/superhuman/brutal_barrage/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
```
This one does not have an `add_attack_logs`

farie82 commented on code/modules/martial_arts/combos/superhuman/foot_skewer.dm:
```diff
@@ -0,0 +1,16 @@
+	explaination_text = "We reform our foot into a serrated blade and stab it deep into our opponent's foot, eviscerating tendons and ligaments and crippling their mobility."
+	steps = list(MARTIAL_COMBO_STEP_HARM, MARTIAL_COMBO_STEP_GRAB)
+
+/datum/martial_combo/superhuman/foot_skewer/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
```
This one does not have an `add_attack_logs`

farie82 commented on code/modules/martial_arts/superhuman.dm:
```diff
@@ -0,0 +1,51 @@
+	return TRUE
+
+/datum/martial_art/superhuman/explaination_header(user)
+	to_chat(usr, "<b><i>We scour the memories of all those we have absorbed, remembering their fighting styles... </i></b>")
```
```suggestion
	to_chat(user, "<b><i>We scour the memories of all those we have absorbed, remembering their fighting styles... </i></b>")
```

farie82 reviewed (changes requested):
Got some minor code things.
Glad to see somebody making new martial arts. It does look cool I gotta admit.

I do have a feeling that this might be a bit strong though. Especially the fact it is hidden till used and always activatable in combination with the burst like combos.
Maybe add more distinguishable effects to the attacks for players to pick up on? Or add effects when the stance is on.

farie82 commented on code/modules/martial_arts/combos/superhuman/brutal_barrage.dm:
```diff
@@ -0,0 +1,17 @@
+/datum/martial_combo/superhuman/brutal_barrage
+	name = "Brutal Barrage"
+	explaination_text = "We unleash a flurry of swift blows that completely decimates an oppenent in a critical state."
+	steps = list(MARTIAL_COMBO_STEP_HARM, MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_HARM)
```
One more small thing. Having grab and then something afterwards will give a wrong help text. Since using grab and then harm on somebody will do nothing. Since you now have a grab as your hand. Bit annoying thing and people will get confused why it is not working while they follow the instructions to the point

Dashsora commented:
Overall, I absolutely love it, and it sounds terribly fun, but I think the instant aggressive grab raises some concerns. 

It renders tentacle kind of pointless, as, yes, it's ranged, but it's also extremely unreliable. I would very often rather attempt to close in the gap, or I'm in melee range already. It doesn't have the rest of its utility, but it has *so much more*, and it doesn't restrict you from using long range weapons either.

I've already told Charlie my suggestion for this, but in the interest to further the conversation here as well, I'd suggest making getting an aggressive grab out of a combo - the death hug.

Help intent - Grab. "We disguise our approach with a hug !"

Not only is it more healthy mechanically, I think it's also terribly thematic to clings as being antags who get you to fear everyone around you. Once there are clings announced on station, watch everyone start to be scared of the hug of death !

hal9000PR commented:
Update: removed the block chance from the martial art
Added a line of text when you examine someone, when one tile away from them, who knows the martial art, denoting that they have it `Their muscles seem to buldge and shift in an inhuman manner.` 

Hopefully this clears up some of @Kyep 's concerns about it not being visible enough. Now at any point a random person can examine you if you pick this and you get called out as a cling, similar to cling eyes and flashes. So it might not be as blatant as arm blade when it is actually being used (even though it is rather obvious, normal people don't tend to claw and smash people) but it is constantly semi-visible. So picking this option is a risk, you have to always position yourself more than one tile away from people to not be examined and given away. AND it costs more than armblade in DNA points.

Sirryan2002 commented on code/game/gamemodes/changeling/powers/superhuman_physiology.dm:
```diff
@@ -0,0 +1,15 @@
+/datum/action/changeling/superhuman
+	name = "Superhuman Physiology"
+	desc = "We combine the fighting knowledge of the hivemind and our enhanced biological strength to form a potent martial art that makes use of our unique abilty to create weapons."
```
```suggestion
	desc = "We utilize the pooled knowledge of the hivemind and our enhanced physiology to further weaponize our unique abilities into a dynamic fighting style."
```

Sirryan2002 commented on code/modules/martial_arts/combos/superhuman/foot_skewer.dm:
```diff
@@ -0,0 +1,17 @@
+/datum/martial_combo/superhuman/foot_skewer
+	name = "Foot Skewer"
+	explaination_text = "We reform our foot into a serrated blade and stab it deep into our opponent's foot, eviscerating tendons and ligaments and crippling their mobility."
```
```suggestion
	explaination_text = "We reform our leg into a serrated blade and jab it deep into our opponent's foot, piercing tendons and ligaments, crippling their mobility."
```
better phrasing and vocab

Sirryan2002 commented on code/modules/martial_arts/combos/superhuman/foot_skewer.dm:
```diff
@@ -0,0 +1,17 @@
+	if(!target.stat && !target.IsWeakened())
+		user.do_attack_animation(target, ATTACK_EFFECT_CLAW)
+		playsound(get_turf(target), 'sound/weapons/slash.ogg', 50, TRUE, -1)
+		target.visible_message("<span class='warning'>[user] reforms their foot into a blade and stabs it into [target]'s foot!</span>",
```
```suggestion
		target.visible_message("<span class='warning'>[user] reforms their foot into a blade and jabs it deep into [target]'s foot!</span>",
```

Sirryan2002 commented on code/modules/martial_arts/combos/superhuman/foot_skewer.dm:
```diff
@@ -0,0 +1,17 @@
+		user.do_attack_animation(target, ATTACK_EFFECT_CLAW)
+		playsound(get_turf(target), 'sound/weapons/slash.ogg', 50, TRUE, -1)
+		target.visible_message("<span class='warning'>[user] reforms their foot into a blade and stabs it into [target]'s foot!</span>",
+						  "<span class='userdanger'>[user] stabs you in the foot, crippling your ability to walk!</span>")
```
```suggestion
						  "<span class='userdanger'>[user] reforms [user.p_their()] foot into a blade and sinks it deep into your foot, crippling your ability to walk!</span>")
```

Sirryan2002 commented on code/modules/martial_arts/combos/superhuman/hammer_fist.dm:
```diff
@@ -0,0 +1,18 @@
+/datum/martial_combo/superhuman/hammer_fist
+	name = "Hammer Fist"
+	explaination_text = "We morph our fist into a bone hammer and slam our target to the ground. The damage is doubled against already stunned targets."
```
```suggestion
	explaination_text = "We morph our fist into a bone hammer and slam our target into the ground. Already stunned targets receive twice as much damage."
```

Sirryan2002 commented on code/modules/martial_arts/combos/superhuman/piston_kick.dm:
```diff
@@ -0,0 +1,18 @@
+		user.apply_damage(5, BURN, pick(BODY_ZONE_R_LEG, BODY_ZONE_L_LEG)) //your leg is literally exploding like a piston
+		var/atom/throw_target = get_edge_target_turf(target, user.dir)
+		target.throw_at(throw_target, 10, 14, user)
+		target.visible_message("<span class='warning'>[user]'s lower leg is explosively propelled into [target], sending them flying!</span>",
```
```suggestion
		target.visible_message("<span class='warning'>[user]'s lower leg is explosively propelled into [target], sending [target.p_them()] flying!</span>",
```

Sirryan2002 commented on code/modules/martial_arts/combos/superhuman/piston_kick.dm:
```diff
@@ -0,0 +1,18 @@
+		var/atom/throw_target = get_edge_target_turf(target, user.dir)
+		target.throw_at(throw_target, 10, 14, user)
+		target.visible_message("<span class='warning'>[user]'s lower leg is explosively propelled into [target], sending them flying!</span>",
+						"<span class='userdanger'>[user]'s lower leg is explosively propelled into you! Sending you flying and disorienting you!</span>")
```
```suggestion
						"<span class='userdanger'>[user]'s lower leg is explosively propelled into you, disorienting you and sending you flying!</span>")
```

Sirryan2002 commented on code/modules/martial_arts/superhuman.dm:
```diff
@@ -0,0 +1,52 @@
+	A.do_attack_animation(D, ATTACK_EFFECT_DISARM)
+	if(prob(60))
+		if(!D.stat && !D.IsWeakened())
+			D.visible_message("<span class='warning'>[A] slams into [D], knocking them to the ground!</span>",
```
```suggestion
			D.visible_message("<span class='warning'>[A] slams into [D], knocking [D.p_them()] to the ground!</span>",
```

Sirryan2002 commented on code/modules/martial_arts/superhuman.dm:
```diff
@@ -0,0 +1,52 @@
+
+/datum/martial_art/superhuman/teach(mob/living/carbon/human/H, make_temporary)
+	. = ..()
+	to_chat(H, "<span class='sciradio'> We shift our biology, we are now able to rapidly form weapons out of different limbs after preforming different attacks. \
```
```suggestion
	to_chat(H, "<span class='sciradio'> We alter our biology, we are now able to rapidly form weapons out of different limbs when preforming special attacks. \
```
better phrasing

Sirryan2002 commented on code/modules/martial_arts/superhuman.dm:
```diff
@@ -0,0 +1,52 @@
+/datum/martial_art/superhuman/teach(mob/living/carbon/human/H, make_temporary)
+	. = ..()
+	to_chat(H, "<span class='sciradio'> We shift our biology, we are now able to rapidly form weapons out of different limbs after preforming different attacks. \
+				Our basic punches are augmented to be slashing claws that do large damage at the cost of knockdown chance. Our disarms become aggressive slams that ram our target to the floor. <span/>")
```
```suggestion
				When punching, our fingers become slashing claws that shred flesh. However, it is at the cost of lower knockdown chance. Our disarms aggressively slam opponents into the floor. <span/>")
```

S34NW commented on code/modules/martial_arts/superhuman.dm:
```diff
@@ -0,0 +1,52 @@
+
+/datum/martial_art/superhuman/teach(mob/living/carbon/human/H, make_temporary)
+	. = ..()
+	to_chat(H, "<span class='sciradio'> We alter our biology, we are now able to rapidly form weapons out of different limbs when preforming special attacks. \
```
```suggestion
	to_chat(H, "<span class='sciradio'> We alter our biology, we are now able to rapidly form weapons out of different limbs when performing special attacks. \
```

farie82 commented on code/modules/martial_arts/combos/superhuman/piston_kick.dm:
```diff
@@ -0,0 +1,18 @@
+	steps = list(MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_GRAB)
+
+/datum/martial_combo/superhuman/piston_kick/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(user != target)
+		user.do_attack_animation(target, ATTACK_EFFECT_KICK)
+		user.apply_damage(5, BURN, pick(BODY_ZONE_R_LEG, BODY_ZONE_L_LEG)) //your leg is literally exploding like a piston
+		var/atom/throw_target = get_edge_target_turf(target, user.dir)
+		target.throw_at(throw_target, 10, 14, user)
+		target.visible_message("<span class='warning'>[user]'s lower leg is explosively propelled into [target], sending [target.p_them()] flying!</span>",
+						"<span class='userdanger'>[user]'s lower leg is explosively propelled into you, disorienting you and sending you flying!</span>")
+		add_attack_logs(user, target, "Melee attacked with martial-art [src] : Piston Kick", ATKLOG_ALL)
+		target.Confused(5)
+		target.apply_damage(10, BRUTE)
+		playsound(get_turf(target), 'sound/weapons/punchmiss.ogg', 80, TRUE, -1)
+		return MARTIAL_COMBO_DONE
```
```suggestion
	if(user == target)
		return MARTIAL_COMBO_FAIL
	user.do_attack_animation(target, ATTACK_EFFECT_KICK)
	user.apply_damage(5, BURN, pick(BODY_ZONE_R_LEG, BODY_ZONE_L_LEG)) //your leg is literally exploding like a 	piston
	var/atom/throw_target = get_edge_target_turf(target, user.dir)
	target.throw_at(throw_target, 10, 14, user)
	target.visible_message("<span class='warning'>[user]'s lower leg is explosively propelled into [target], sending [target.p_them()] flying!</span>",
					"<span class='userdanger'>[user]'s lower leg is explosively propelled into you, disorienting you and sending you flying!</span>")
	add_attack_logs(user, target, "Melee attacked with martial-art [src] : Piston Kick", ATKLOG_ALL)
	target.Confused(5)
	target.apply_damage(10, BRUTE)
	playsound(get_turf(target), 'sound/weapons/punchmiss.ogg', 80, TRUE, -1)
	return MARTIAL_COMBO_DONE
```
Best to apply an early return here and explicitly state that the combo failed

farie82 reviewed (commented):
Minor thing

variableundefined commented:
No head or maint approval in 90+ days. Closing.
